entity-service-client/src/main/java/org/hypertrace/entity/data/service/client/EdsTypeAndIdAttributesCacheKey.java

```diff
@@ -0,0 +1,40 @@
+package org.hypertrace.entity.data.service.client;
+
+import java.util.Objects;
+import org.hypertrace.entity.data.service.v1.ByTypeAndIdentifyingAttributes;
+
+public class EdsTypeAndIdAttributesCacheKey {
```

Comment:
Unit test to verify the equals of different objects?

Reply:
Done.